DOC-763: Add notification / editor blocking behaviour to the 5.7 release notes (#1863)
tinymce/tinymce-docs: release-notes/release-notes57.md

The following enhancements were made for the {{site.productname}} 5.7 release.

### Changes to notifications and popups when the editor is blocked

> Notifications opened with the Notifications API will now remain visible when the editor's progress state is set to `true` using the {{ site.productname }} `setProgressState()` API. Additionally, popups within the editor will now be closed while the editor's progress state is set to `true`.

Reviewer: considering we had to clarify amongst ourselves what popups were, maybe some examples would be good? not sure. I don't know if/where we use "popups" as a term elsewhere.

Author: We did consider adding a "For information about popups see ..." but then realised there was nowhere in the docs that we explained popups as a concept. Do you think we should Jira up adding that (and then updating this release note)?

Reviewer 2: maybe 

> popups within the editor will ...

to 

> popups within the editor (such as thing popup and otherthing popup) will ...

(obviously replacing thing and other thing) (reminder: don't say- eg, ie, or etc)

> For information on the {{ site.productname }} `setProgress()` API, see [Editor API - `setProgressState`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate).

Reviewer 2: Restruct the "for infos" pls

For information on:

- thing one
- thing two

The following enhancements were made for the {{site.productname}} 5.7 release.

> ### Notifications are now shown when the editor is blocked

Reviewer 3: ```suggestion
### Changes to notifications and popups when the editor is blocked
```
This is not about notifications, it's about the changes that happen when the progress state is set to true. So I've just reworded it to reflect that.

> Notifications opened with the Notifications Api (`NotificatonApi`) will remain visible when the editor's progress state is set to true using the {{ site.productname }} API [`tinymce.activeEditor.setProgressState()`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate). Additionally, popups within the editor will be closed while the editor's progress state is set to true. For information on the Notifications API see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).

Reviewer 4: ```suggestion
Notifications opened with the Notifications API (`NotificationApi`) will remain visible when the editor's progress state is set to `true` using the {{ site.productname }} API [`tinymce.activeEditor.setProgressState()`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate). Additionally, popups within the editor will be closed while the editor's progress state is set to `true`. For information on the Notifications API see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).
```

Should it also be ```(`NotificationAPI`)```?

> Notifications opened with the Notifications Api (`NotificatonApi`) will remain visible when the editor's progress state is set to true using the {{ site.productname }} API [`tinymce.activeEditor.setProgressState()`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate). Additionally, popups within the editor will be closed while the editor's progress state is set to true. For information on the Notifications API see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).

Reviewer 3: ```suggestion
Notifications opened with the Notifications API will now remain visible when the editor's progress state is set to true using the {{ site.productname }} `setProgressState()` API. Additionally, popups within the editor will be closed while the editor's progress state is set to true.

For information on the editor `setProgress()` API, see [Editor API - `setProgressState`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate).

For information on the Notifications API, see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).
```

This needs breaking up a little and we shouldn't mention `NotificationApi` as that's confusing. I did originally work out that part in the brackets was referring to the type, but types aren't documented and isn't something you'd find in a sentence normally anyways.
